Integrations/integration-Netskope.yml

```diff
@@ -225,13 +225,45 @@ script:
         return {
             Type: entryTypes.note,
             ContentsFormat: formats.json,
-            Contents: retArray,
+            Contents: result.data,
             ReadableContensFormat: formats.markdown,
             HumanReadable: tableToMarkdown('Netskope Events', retArray, headers),
             EntryContext: ec
         };
     }
 
+    function now() {
+        return parseInt(new Date().getTime()/1000)
+    }
+
+    function fetchAlerts() {
+        last_run = parseInt(getLastRun())
```

Comment:
you are missing many `var`'s and `;`'s

Reply:
js sucks.

Reply:
Use powershell instead

Reply:
most of the code was mising it.. fixed it for you!